resources/zib/zib-BodyHeight.xml

```diff
@@ -0,0 +1,133 @@
+        <map value="NL-CM:12.2.4" />
+        <comment value="HeightDateTime" />
+      </mapping>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]" />
+      <short value="HeightValue" />
+      <definition value="The element contains the value of the measured body height.  &#xD;&#xA;Body height is the height of the human body from heel to scalp, generally measured when standing. Body height is measured in a stretched, horizontal position for very small children or patients that are unable to stand." />
+      <alias value="LengteWaarde" />
+      <min value="1" />
+      <type>
+        <code value="Quantity" />
+      </type>
+      <patternQuantity>
+        <system value="http://loinc.org" />
+        <code value="8302-2" />
+      </patternQuantity>
+      <mapping>
+        <identity value="zib-bodyheight-v3.1.1-2020EN" />
+        <map value="NL-CM:12.2.2" />
+        <comment value="HeightValue" />
+      </mapping>
+    </element>
+    <element id="Observation.note">
+      <path value="Observation.note" />
+      <short value="Comment" />
+      <definition value="Comment on the body height measurement." />
+      <alias value="Toelichting" />
+      <max value="1" />
+      <mapping>
+        <identity value="zib-bodyheight-v3.1.1-2020EN" />
+        <map value="NL-CM:12.2.3" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.component:position">
+      <path value="Observation.component" />
+      <sliceName value="position" />
+      <short value="Position" />
+      <definition value="The person's position during the measurement." />
+      <alias value="Positie" />
+      <max value="1" />
+      <mapping>
+        <identity value="zib-bodyheight-v3.1.1-2020EN" />
+        <map value="NL-CM:12.2.5" />
+        <comment value="Position" />
+      </mapping>
+    </element>
+    <element id="Observation.component:position.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="425260001" />
+          <display value="Body position for height measurement" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:position.code.coding">
+      <path value="Observation.component.code.coding" />
+      <binding>
+        <strength value="extensible" />
+        <description value="PositieCodelijst" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.12.2.1--20200901000000" />
```

Comment:
Dit staat op de verkeerde plek, dient op .component.value[x] te komen.

Reply:
Moved the codeList to value[x]